### DISCUSSION TITLE
Boosts Information - Compact Display doesn't line up very well

### DISCUSSION THREAD

**Discussion by robuuuste** [runelite/runelite#15686] [Ideas]
The compact display option for the Boosts Information plugin is very nice but I think it could look better since the boosts don't line up very well. Would it be possible to make them align properly?

Example of what it currently looks like:
![abcd](https://user-images.githubusercontent.com/91341050/190809067-e15971a4-6a32-4935-9e1f-5bca3fa2a3d8.png)

How I think it would look better:
![abc](https://user-images.githubusercontent.com/91341050/190809027-0187fff9-9fa4-4f3a-8fa9-e05aa83f8d8f.png)

**Comment by geheur** [comment]
The ranged icon is 2 px wider than the rest (it is 18x18 and the others are 16x16). fwiw the "overall" icon is 2px smaller (14x14), although I don't think it's used in the compact display. The ranged icon being off by 2 px is what makes the first screenshot look bad.

**Reply by robuuuste** [reply]
> The ranged icon is 2 px wider than the rest (it is 18x18 and the others are 16x16). fwiw the "overall" icon is 2px smaller (14x14), although I don't think it's used in the compact display. The ranged icon being off by 2 px is what makes the first screenshot look bad.

I see. I might be wrong but I guess if you made all the icons the same size, for example a 20 x 20px with a transparent background, that should fix it right? It wouldn't resize the visual part of the icon, only the transparent border.
